packages/gel/src/options.ts

```diff
@@ -427,7 +432,8 @@ export class Options {
       this.config.size === 0 &&
       this.globals.size === 0 &&
       this.moduleAliases.size === 0 &&
-      this.module === "default"
+      this.module === "default" &&
+      this.transactionOptions === TransactionOptions.defaults()
```

Comment:
I don't like this comparison -- it's kinda weird because there are multiple ways how you can end up manually configuring `transactionOptions` to be in the "default" state and not match here with `===`.

Please add `TransactionOptions.isDefault()` method and call it here, checking against concrete defaults inside.

Reply:
I think the `.defaults()` singleton must go, there is no use for it and allowing it to even exist in our code base was a mistake.

Reply:
The new "defaults" is now an empty object, but good call that it won't always be reference equal here. I'll fix that up.